datacite-doi-commons/src/test/java/no/unit/nva/doi/model/DoiUpdateDtoTest.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.doi.model;
+
+import static no.unit.nva.hamcrest.DoesNotHaveNullOrEmptyFields.doesNotHaveNullOrEmptyFields;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.time.Instant;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.Test;
+
+public class DoiUpdateDtoTest {
+
+    private final ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    @Test
+    public void canWriteDoiUpdateDtoToJsonAndBack() throws JsonProcessingException {
+        DoiUpdateDto doiUpdateDto = new DoiUpdateDto.Builder()
+            .withDoi(URI.create("http://sample.doi"))
```

Comment:
Want this ideally to be 

```
private static final String DEMO_PREFIX = "10.5072";
private static final URI STAGE_DOI_PROXY = URI.create("https://handle.stage.datacite.org/");
private static final String EXAMPLE_PREFIX = DEMO_PREFIX;
private static final String EXAMPLE_SUFFIX = createRandomSuffix() // UUID.random.toString()
private static final URI EXAMPLE_PROXY = STAGE_DOI_PROXY;
private static final String EXAMPLE_IDENTIFIER = EXAMPLE_PREFIX + FORWARD_SLASH + EXAMPLE_SUFFIX;
public static final URI EXAMPLE_DOI = URI.create(EXAMPLE_PROXY + EXAMPLE_IDENTIFIER);

public URI createExampleDoi() {
  return Doi.builder().withDoi(EXAMPLE_DOI).build().toUri()
}

```
(This tells the story about how the DOI looks like, and also ensures it has valid values.)

But it kinds depends https://github.com/BIBSYSDEV/nva-publication-api/pull/96 for using the `Doi.builder`, so you could use `EXAMPLE_DOI` directly, but missing out on the extra validation for `Doi`